packages/ui-file-drop/src/FileDrop/index.tsx

```diff
@@ -169,7 +169,7 @@ class FileDrop extends Component<FileDropProps, FileDropState> {
       list = (this.props as any).shouldAllowMultiple ? list : [list[0]]
     }
     if (shouldEnablePreview) {
-      return list.map((file) =>
+      return list.map((file: any) =>
```

Comment:
This threw a new error, but since this component is not typed I just left it as `any`